### ISSUE TITLE
Why the service page is empty?

### ISSUE THREAD

**Issue by dawnyesky** [huginn/huginn#2825]
I followed the official manual to deploy the docker images on my server. But the service page is initially empty. Is it normal? I also follow the [wiki](https://github.com/huginn/huginn/wiki/Configuring-OAuth-applications) to set the API key for twitter but nothing changes. Is there something I missed during setup? I could not find any documentation about this issue.

**Comment by degan6** [comment]
For services to work, you have to add a twitter client ID and secret to the config file. Have you done that?

**Comment by dawnyesky** [comment]
No. I didn't see this instruction on the docker manual. Could you include a link here? But I think using a docker is not supposed to modify the internal files. Is it a good idea to map the modified config file to the docker while running?

------------------ Original ------------------

**Comment by dsander** [comment]
Services are configured using [environment variables](https://github.com/huginn/huginn/blob/master/.env.example#L131-L157), when using docker those variables are not specified in the `.env` file but passed to the container then it is started.
